src/main/kotlin/lotto/LottoSeller.kt

```diff
@@ -14,6 +14,11 @@ class LottoSeller(private val money: Money) {
         return money.divide(LOTTO_PRICE).won
     }
 
+    fun buyManual(count: Int) {
+        require(getLottoCount() >= count) {"해당 금액보다 초과된 로또 갯수입니다."}
```

Comment:
```suggestion
        require(getLottoCount() >= count) { "해당 금액보다 초과된 로또 갯수입니다." }
```
ktlint를 통과하지 못하고 있어요.